siege_utilities/reporting/analytics/polling_analyzer.py

```
@@ -108,261 +106,130 @@ def create_longitudinal_analysis(self,
+                (merged[f'{metric}_current'] - merged[f'{metric}_historical']) /
                 merged[f'{metric}_historical'] * 100
             ).fillna(0)
-            
+
             # Add change direction
             merged['change_direction'] = merged['change_pct'].apply(
                 lambda x: 'growth' if x > 0 else 'decline' if x < 0 else 'stable'
             )
-            
+
             return merged
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)